pkg/users/local.js

```diff
@@ -81,26 +83,38 @@ function AccountsPage() {
     );
 
     if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
-    } else {
+        return (
+            <AccountsMain
+                accountsInfo={accountsInfo}
+                current_user={current_user_info && current_user_info.name}
+                groups={groupsExtraInfo || []}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230124-132122-e115a2b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)